src/PerformanceSensitiveAnalyzers/PerformanceSensitiveAnalyzers.md

```diff
@@ -1 +1,41 @@
-This is forked from [01c6dd2bf7dc509de289da60b78df8440ce4c11d@Microsoft/RoslynClrHeapAllocationAnalyzer](https://github.com/Microsoft/RoslynClrHeapAllocationAnalyzer/commit/01c6dd2bf7dc509de289da60b78df8440ce4c11d)
\ No newline at end of file
+This is forked from [01c6dd2bf7dc509de289da60b78df8440ce4c11d@Microsoft/RoslynClrHeapAllocationAnalyzer](https://github.com/Microsoft/RoslynClrHeapAllocationAnalyzer/commit/01c6dd2bf7dc509de289da60b78df8440ce4c11d)
+
+How to avoid conflicts of `PerformanceSensitiveAttribute` in projects contain `InternalsVisibleTo` (IVT)
+--------------------------------
+
+Because we inject the source of `PerformanceSensitiveAttribute` into projects referencing PerformanceSensitive analyzer package by default, which is declared as `internal`, you may run into warning CS0436 if you have IVTs defined in your projects.
```

Comment:
```suggestion
Because we inject the source of `PerformanceSensitiveAttribute` into projects referencing `PerformanceSensitive` analyzer package by default, which is declared as `internal`, you may run into warning CS0436 if you have IVTs defined in your projects.
```

Reply:
I'm using quote for code only.